### PR TITLE
Enable additional currencies

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

**Comment by ykris45** [comment]
thanks you so much for your assistance, i referenced my PR here #755

**Comment by mvaivre** [comment]
The last 3 currencies make the wallet crash (I guess they are not available on coin gecko?)

We should remove them.

But good news: testing this made me find a little issue: the currency ticker isn't updated in the greeting message showing the current ALPH price :)

**Comment by ykris45** [comment]
@mvaivre RUB, VND and Also TRY is available on Coingecko

**Comment by ykris45** [comment]
image
image
image
see my screenshot here

**Comment by nop33** [comment]
@mvaivre ha, weird! I Guess it worked for me cause I had the wallet already running, after restarting it I noticed the problem as well! @ykris45 there's nothing for you to worry about, it's our problem 😉

File: src/pages/UnlockedWallet/OverviewPage/GreetingMessages.tsx [hunk 1]
```diff
@@ -48,7 +48,8 @@ const GreetingMessages = ({ className }: GreetingMessagesProps) => {
 
   const priceComponent = (
     <span key="price">
-      {price ? '📈 ' + t('ALPH price: {{ price }}$', { price: formatFiatAmountForDisplay(price) }) : ''}
+      {price ? '📈 ' + t('ALPH price: {{ price }}', { price: formatFiatAmountForDisplay(price) }) : ''}
+      {currencies[fiatCurrency].symbol}
```
Comment (nop33) [inline src/pages/UnlockedWallet/OverviewPage/GreetingMessages.tsx:52]: hm, so, if there is no `price`, then just the symbol is being displayed?

Comment (nop33) [reply src/pages/UnlockedWallet/OverviewPage/GreetingMessages.tsx:52]: I think it'd be better to do

```ts
{price ? '📈 ' + t('ALPH price: {{ price }}', { price: formatFiatAmountForDisplay(price) }) + currencies[fiatCurrency].symbol : ''}
```

or a bit cleaner (imo)

```ts
{price ? `📈  ${t('ALPH price: {{ price }}', { price: formatFiatAmountForDisplay(price) })} ${currencies[fiatCurrency].symbol}` : ''}
```

Comment (mvaivre) [reply src/pages/UnlockedWallet/OverviewPage/GreetingMessages.tsx]: 💜

Comment (mvaivre) [reply src/pages/UnlockedWallet/OverviewPage/GreetingMessages.tsx]: (I didn't see your previous comment before commiting)

**Pull request merged**